### PR TITLE
fix(tue-checkout): Also checkout .tue dir

### PR DESCRIPTION
**Pull request by reinzor**

### CODE REVIEW

File: setup/tue.bash_functions [hunk 1]
```diff
@@ -568,6 +568,7 @@ function tue-checkout
     fi
 
     local branch=$1
+	cd $TUE_DIR; git checkout $branch > /dev/null; cd - > /dev/null
```
Comment (MatthijsBurgh) [inline setup/tue.bash_functions]: Please use same implementation as pkgs

**Comment by reinzor** [comment]
Why?

-Rein

**Comment by MatthijsBurgh** [comment]
At least the visuals would be useful

**Comment by MatthijsBurgh** [comment]
I want to extend this with the option to disable the checkout of the .tue dir.

**Comment by MatthijsBurgh** [comment]
@Timple could you create an option to not checkout the .tue dir? Like --only-pkgs.

**Comment by MatthijsBurgh** [comment]
@Timple please review

File: setup/tue.bash_functions [hunk 2]
```diff
@@ -561,18 +561,43 @@ function tue-checkout
     then
         echo """Switches all packages to the given branch, if such a branch exists in that package. Usage:
 
-    tue-branch BRANCH-NAME
+    tue-checkout BRANCH-NAME [option]
+
+    options:
+    --only-pks: tue-env is not checkout to the specified branch
```
Comment (LoyVanBeek) [inline setup/tue.bash_functions]: *check**ed** out

**Pull request merged**